radix-engine/src/engine/precommitted_kv_store.rs

```diff
@@ -1,16 +1,100 @@
 use sbor::rust::cell::{RefCell, RefMut};
+use sbor::rust::collections::hash_map::IntoIter;
 use sbor::rust::collections::*;
 use sbor::rust::vec::Vec;
 use scrypto::engine::types::*;
 use scrypto::values::ScryptoValue;
 
 use crate::model::*;
 
+#[derive(Debug)]
+pub struct InMemoryChildren {
+    child_values: HashMap<StoredValueId, RefCell<StoredValue>>,
```

Comment:
Using `StoredValueId` for in-memory children looks odd. How about `PersistableValueId` and `TransientValueId`?